circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/util/LibJarDeployer.java

```diff
@@ -42,15 +43,32 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(LibJarDeployer.class);
 
+  private static class PriviledgedClassLoader implements PrivilegedAction<Void> {
```

Comment:
What the reason for this one? The interface javadoc seems to not really justify the usage here

Reply:
Ah, yes, I should've added a comment for this:

```
com.hotels.bdp.circustrain.core.util.LibJarDeployer.libjars(Configuration, Class[]) 
creates a java.net.URLClassLoader classloader, which should be performed within 
a doPrivileged block.

This code creates a classloader, which needs permission if a security manage is 
installed. If this code might be invoked by code that does not have security 
permissions, then the classloader creation needs to occur inside a doPrivileged 
block.
```

Reply:
Interesting, sounds like a reasonable change, we just need to make sure this still works for real when run on a cluster etc.

Reply:
I don't think adding an interface like we have now makes it run in a privileged block I think you then need to call it like this???
```
 AccessController.doPrivileged(new PriviledgedClassLoader(conf, libjars));
```

Reply:
Yes, you're right! I missed that part of it.